docs/overview.md

```diff
@@ -0,0 +1,844 @@
+# Introduction
+
+## Overview
+
+The MongoDB Atlas SQL ODBC driver supports connections from a SQL compliant
+client, enabling you to query your data in MongoDB with SQL queries.
+
+### Features
+
+Key Features: Native object and array support.
+
+Compatibility: The MongoDB Atlas SQL ODBC driver is compatible with MongoDB versions greater
+than or equal to 6.0.6.
+
+## System Requirements
+
+### Hardware Requirements
+
+The MongoDB Atlas SQL ODBC driver has no hard system requirements itself. Any system
+capable of running a modern SQL tool (such as Power BI, DBeaver) is capable of running
+the MongoDB Atlas SQL ODBC driver.
+
+### Software Requirements
+
+#### Supported Operating Systems
+
+The MongoDB ODBC driver is compatible with Windows x86_64 architecture, linux x86_64,
+and linux arm2 systems.
```

Comment:
What do you mean by `arm2 systems`?  I usually see arm64 or maybe amazon linux 2 (ARM)